src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -44,6 +44,7 @@
 #include "NewProject/ProjectManager/project_manager.h"
 #include "Utils/FileUtils.h"
 #include "Utils/LogUtils.h"
+#include "Utils/ScopeGuard.hpp"
```

Comment:
Why is it not in third_party ?
https://github.com/os-fpga/FOEDAG/tree/main/third_party

Reply:
I was just thinking about it as a util and third party didn't occur to me. I'll move it.